govc/version/command.go

```diff
@@ -36,8 +36,8 @@ type version struct {
 
 func init() {
 	// Check that git tag in the release builds match the hardcoded version
-	if gitVersion != "" && gitVersion[1:] != flags.Version {
-		log.Panicf("version mismatch: git=%s vs govc=%s", gitVersion[1:], flags.Version)
+	if gitVersion != "" && gitVersion != flags.Version {
```

Comment:
`flags.Version` doesn't have the `v` prefix, but the git tag does (existing ones do at least).  We could also just remove this check.  The check was added when we added the hard-coded `flags.Version`.  I'd be ok removing `flags.Version`, only added it since `go get` does not inject gitVersion, the hardcoded version at least gave a hint what the `go get` folks were using.